pkg/runtime/wfengine/component.go

```diff
@@ -141,7 +138,7 @@ func (c *workflowEngineComponent) Purge(ctx context.Context, req *workflows.Purg
 
 	if err := c.client.PurgeOrchestrationState(ctx, api.InstanceID(req.InstanceID)); err != nil {
 		if errors.Is(err, api.ErrInstanceNotFound) {
-			c.logger.Infof("Unable to purge the instance: '%s', no such instance exists", req.InstanceID)
+			c.logger.Errorf("Unable to purge the instance: '%s', no such instance exists", req.InstanceID)
```

Comment:
At maximum, I think this should just be a warning. Error to me feels like something with Dapr is broken, which is definitely not the case here.

Reply:
Will change

Reply:
```suggestion
			c.logger.Warnf("Unable to purge the instance: '%s', no such instance exists", req.InstanceID)
```